lua/loc/fr.json

```diff
@@ -97,27 +120,38 @@
   "bm_wp_ass_s552_fg_standard_green" : "Garde-main OD Green",
   "bm_wp_ass_s552_fg_railed"         : "Garde-main à rail SIG SG 553",
   "bm_wp_ass_s552_b_long"            : "Kit SIG SG 552 LB",
+	"bm_wp_ass_s552_g_standard_green"  : "Poignée OD Green",
+	"bm_wp_ass_s552_s_standard_green"  : "Crosse OD Green",
 
   "bm_wp_aug_fg_a3"    : "Poignée avant AUG A3",
 	"bm_wp_aug_body_f90" : "Boite de culasse Thales EF88/F90",
 	"bm_wp_aug_b_short"  : "Canon 16 pouces  Carabine AUG",
+	"bm_wp_aug_b_long"   : "Canon AUG HBAR",
 
   "bm_wp_l85a2_fg_short" : "Poignée avant à rail Daniel Defense",
   "bm_wp_l85a2_b_short"  : "Canon Prototype L22",
+	"bm_wp_l85a2_b_long"   : "Canon L86A1",
+	"bm_wp_l85a2_g_worn"   : "Poignée abimée",
 
   "bm_wp_famas_b_suppressed" : "Canon entièrement surpressé ",
 	"bm_wp_famas_b_long"       : "Canon civil G2",
 	"bm_wp_famas_b_short"      : "Canon G2 Commando",
-	"bm_wp_famas_b_sniper"     : "Canon G2 Sniper",
+	"bm_wp_famas_b_sniper"     : "Canon G2 de tir de précision",
```

Comment:
I can technically let Sniper, but i hate when people just casually call those rifles as snipers. I mean, It god the benefice to be atleast easy to say, but also to make them be called by a wrong abriveation and calling other simple semi auto weapons as snipers aswell, just because they got a bigger round size then the other classical assault weapons.
So, Here, Translated accurately, "Sharpshotter" as "Tireur de précision" (As, a precision rifle).